### PR TITLE
Remix bars

### PR DESCRIPTION
**Pull request by lankaukk**
Several fixes to the design of the Scene Title Bars and the Toolbar's Play Mode subsection:

Scene Title Bars:
- reordering the contents
- all the same height
- same distance from the scene itself
- better background colors (aqua, purple, and grey) for different types of scenes in both dark and light mode
- back and forwards buttons hidden when scene is not selected
- castle home button, because obviously ⛫

Play Mode Toolbar:
- better font sizes
- fixing dark mode colors
- button color change to dynamic blue when hovering
- castle home button ⛫

Screenshot 2023-09-21 at 3 42 16 PM
Screenshot 2023-09-21 at 3 42 31 PM

Screenshot 2023-09-21 at 3 43 39 PM
Screenshot 2023-09-21 at 3 43 26 PM

### CODE REVIEW

File: editor/src/components/canvas/controls/select-mode/remix-scene-label.tsx [hunk 1]
```diff
@@ -243,14 +255,13 @@ const RemixSceneLabel = React.memo<RemixSceneLabelProps>((props) => {
...
+          color: colorTheme.aqua.value,
           position: 'absolute',
           left: frame.x,
           bottom: -frame.y + offsetY,
           width: frame.width,
-          paddingLeft: offsetX,
-          paddingTop: paddingY,
-          paddingBottom: paddingY,
+          height: scaledLineHeight * 2,
+          padding: '10px 20px',
```
Comment (maltenuhn) [inline editor/src/components/canvas/controls/select-mode/remix-scene-label.tsx:264]: I think these need to be zoom-safe, so I'd either restore the previous code or adapt it

File: editor/src/components/canvas/controls/select-mode/remix-scene-label.tsx [hunk 2]
```diff
@@ -243,14 +255,13 @@ const RemixSceneLabel = React.memo<RemixSceneLabelProps>((props) => {
...
-          color: isIndexRoute ? colorTheme.aqua.value : colorTheme.textColor.value,
+          color: colorTheme.aqua.value,
           position: 'absolute',
           left: frame.x,
           bottom: -frame.y + offsetY,
           width: frame.width,
-          paddingLeft: offsetX,
-          paddingTop: paddingY,
-          paddingBottom: paddingY,
+          height: scaledLineHeight * 2,
```
Comment (maltenuhn) [inline editor/src/components/canvas/controls/select-mode/remix-scene-label.tsx:263]: visually the new bars are still much too tall and prominent for me. Can we bring this down to something more like 2-3px top/bottom padding around the text?

File: editor/src/components/canvas/controls/select-mode/remix-scene-label.tsx [hunk 3]
```diff
@@ -259,47 +270,70 @@ const RemixSceneLabel = React.memo<RemixSceneLabelProps>((props) => {
...
+            }}
           >
-            〱
-          </span>
-        </Tooltip>
-        <Tooltip title={'Home'}>
+            {label}
+          </div>
+        </FlexRow>
+        <FlexRow style={{ gap: 10 }}>
```
Comment (maltenuhn) [inline editor/src/components/canvas/controls/select-mode/remix-scene-label.tsx:298]: is this zoom-safe?

File: editor/src/components/canvas/controls/select-mode/scene-label.tsx [hunk 4]
```diff
@@ -209,16 +209,14 @@ const SceneLabel = React.memo<SceneLabelProps>((props) => {
...
+            color: sceneHasSingleChild ? colorTheme.componentPurple.value : colorTheme.fg2.value,
             position: 'absolute',
             fontWeight: 600,
             left: frame.x,
             bottom: -frame.y + offsetY,
             width: frame.width,
-            paddingLeft: offsetX,
-            paddingBottom: paddingY,
+            height: scaledLineHeight * 2,
+            padding: '10px 20px',
```
Comment (maltenuhn) [inline editor/src/components/canvas/controls/select-mode/scene-label.tsx:219]: (same zoom safety / height questions here as above)

File: editor/src/components/canvas/controls/select-mode/scene-label.tsx [hunk 5]
```diff
@@ -229,8 +227,15 @@ const SceneLabel = React.memo<SceneLabelProps>((props) => {
             backgroundColor: backgroundColor,
           }}
         >
-          {label}
-        </div>
+          <div
+            style={{
+              padding: `${4 / scale}px`,
+              fontWeight: 'bold',
```
Comment (maltenuhn) [inline editor/src/components/canvas/controls/select-mode/scene-label.tsx:233]: could you use numbers here? it makes searching for those across the repo a lot easier. Also, are you sure this should be bold - I don't think we use 700 anywhere, typically 600 is our max

File: editor/src/components/editor/remix-navigation-bar.tsx [hunk 6]
```diff
@@ -55,40 +58,52 @@ export const RemixNavigationBar = React.memo(() => {
...
         pointerEvents: 'initial',
         userSelect: 'none',
-        padding: 4,
-        color: theme.fg4.value,
       }}
       onMouseDown={stopPropagation}
       onClick={stopPropagation}
     >
-      <Tooltip title={'Back'}>
+      <Tooltip title={'Back'} placement='bottom'>
```
Comment (maltenuhn) [inline editor/src/components/editor/remix-navigation-bar.tsx:70]: nice!

**Review by seanparsons — APPROVED** [review]

**Review by gbalint — APPROVED** [review]

**Pull request merged**